backend/routes/health_api.py

```
@@ -1,7 +1,78 @@
+    except Exception:
+        return "down"
+
+
+def _build_payload(request: Request, redis_status: str, db_status: str) -> dict[str, Any]:
+    return {
+        "status": "ok",
+        "service": "backend",
+        "version": getattr(request.app, "version", "0.0.0"),
+        "redis": redis_status,
+        "db": db_status,
+        "routers_loaded": len(getattr(request.app, "router").routes),
```

Ruff (0.14.14)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)